package-scripts.js

```diff
@@ -44,29 +47,67 @@ module.exports = {
           'build.cjs.tiny',
           'build.umd.main.tiny',
           'build.umd.min.tiny'
-        )
+        ),
+        rimraf('preact'),
+        concurrent.nps(
+          'build.es.preact',
+          'build.cjs.preact',
+          'build.umd.main.preact',
+          'build.umd.min.preact',
+          'build.es.preact.tiny',
+          'build.cjs.preact.tiny',
+          'build.umd.main.preact.tiny',
+          'build.umd.min.preact.tiny'
+        ),
+        copy("'package.json' 'preact'")
```

Comment:
Hmmm... What does this do exactly?

I'm a little confused about the build process. When everything's finished, what are we left with?

Reply:
It creates a directory `preact` in which the are all builds for preact as well as a copy of `package.json` from the root directory. No typings at this time since current type definitions are react-dependent.

Reply:
I see! So, someone using the package would do:

```
import glamorous from 'glamorous/preact'
```

Is that right? That makes good sense. For `downshift` I push everything into the [`dist/`](https://unpkg.com/downshift@1.4.0/dist/) and have `.preact` filename, but I think I like this better. Thanks!